internal-packages/workflow-designer-ui/src/ui/search-filter.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from "react";
```

Comment:
Is this a sample code?
If you don't need this file, I would appreciate it if you could delete it.